data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -88,10 +88,8 @@
   "difficulty": 2,
   "time": 10000,
   "book_learn": [["adv_chemistry", 1] , ["textbook_chemistry", 1]],
-  "tools": [
-    [
-      [ "chemistry_set", 5 ]
-    ]
+  "qualities": [
+    {"id":"CHEM","level":1}
```

Comment:
Hydrogen and chlorine gas are the primary outputs, with sodium hydroxide as a by-product. It might additionally be reasonable to assume that the chemistry set has a source of UV light as well, needed for synthesizing hydrogen chloride, which is in turn used to make hydrochloric acid. One could reasonably assume the generic acid is hyrochloric acid, as we already have nitric acid, surfuric acid, and acetic acid as separate items.

https://en.wikipedia.org/wiki/Hydrogen_chloride#Direct_synthesis